cmd/dataobj-inspect/go.mod

```diff
@@ -4,133 +4,28 @@ go 1.23.1
 
 replace github.com/grafana/loki/v3 => ../..
 
-require (
-	github.com/dustin/go-humanize v1.0.1
-	github.com/grafana/loki/v3 v3.3.2
-)
+require github.com/grafana/loki/v3 v3.3.2
```

Comment:
@benclive Go told me I needed to run `go mod tidy`, let me know if anything here seems off